build.package.js

```diff
@@ -0,0 +1,57 @@
+const path = require("path")
+const esbuild = require("esbuild")
+const fs = require("fs")
+const { buildOptions } = require("./build")
+
+async function findFiles(directory, extension, files = []) {
+  await Promise.all(
+    fs.readdirSync(directory).map(async (filePath) => {
+      const absolute = path.join(directory, filePath)
+      if (fs.statSync(absolute).isDirectory()) {
+        await findFiles(absolute, extension, files)
+      } else if (absolute.endsWith(`.${extension}`) && !absolute.endsWith(`.test.${extension}`)) {
+        files.push(absolute)
+      }
+    })
+  )
+
+  return files
+}
+
+async function resolveImportPaths() {
```

Comment:
Unfortunately, I couldn't find a straightforward solution for this. I've written a function that updates paths starting with `src/` by transforming them into relative paths.